.github/workflows/libosrm.yml

```diff
@@ -0,0 +1,61 @@
+name: Libosrm CI
+on:
+  push:
+     branches:
+       - master
+     paths:
+       - '.github/workflows/libosrm.yml'
+       - '**libosrm_wrapper**'
```

Comment:
I would remove the `libosrm_wrapper` path restriction here, in order to systematically check build against `libosrm` for each new feature merge in `master` (that will be cheap enough thanks to caching). If we keep the restriction for `pull_request` we'll filter out most of the unnecessary  work anyway.

Reply:
Removed path restriction on push to master.